var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -448,6 +449,11 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
         sha256="063a38f402439fa39fd8d57315a321e6510adcd04aec5400a88e744aaa60bc8e",
         when="@13.0.0:13.0.1 +teko gotype=long",
     )
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/trilinos/Trilinos/pull/11134.patch?full_index=1",
+        sha256="6d1bb2eb8963ef438ac4b548fb11b5dc1248cb99c3a6a59cd18a0fa966789ec2",
+        when="@13.4.0 +superlu-dist",
```

Comment:
Is this only `@13.4.0` intentional (patch unneeded after 13.4.0 because it was fixed after that) or should it be `@13.4.0:13` like the new use of superlu-dist@8:?

Reply:
@kuberry , the  patch  (to trilinos) is now merged into https://github.com/trilinos/Trilinos/tree/trilinos-release-13-4-branch - so I would think it will be included in `13.4.1` [if one is released]. So I think `@13.4.0` is appropriate here.

And this change to trilinos spec is in a separate PR at https://github.com/spack/spack/pull/33533